.github/workflows/build-app.yml

```diff
@@ -0,0 +1,35 @@
+name: Build App
+
+on:
+    workflow_call:
+        inputs:
+            ref:
+                type: string
+            force_pack:
+                type: boolean
```

Comment:
Wondering where the comes from (at least I couldn't see it being passed in the workflow using this in RSSI)

Reply:
It is used in the release workflow: https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/artifactory/github_actions/.github/workflows/release-app.yml#L20